src/decrypt-api/Startup.cs

```diff
@@ -50,10 +50,7 @@ public Startup(IHostingEnvironment env)
             });
 
             services.AddSingleton<IKubernetes>(s => {
-                var config = string.IsNullOrEmpty(Configuration["Kubernetes:ProxyUrl"])
-                ? KubernetesClientConfiguration.BuildDefaultConfig()
-                : new KubernetesClientConfiguration { Host = Configuration["Kubernetes:ProxyUrl"] };
-                return new Kubernetes(config);
+                return new Kubernetes(KubernetesClientConfiguration.BuildDefaultConfig());

```

Comment:
Isn't relevant to this PR

Reply:
yep I needed it for local testing of the init container... so I thought I'll add it

Reply:
Isn't the ProxyUrl is required for testing?

Reply:
This replaces it - like the CRD - use either configured `KUBECONFIG` or in cluster config in one line. This was added in recent versions of the SDK...

Reply:
NM it breaks the build - will fix it in a latter point.